src/System.Security.Cryptography.Algorithms/tests/HashAlgorithmTest.cs

```diff
@@ -53,6 +53,100 @@ protected void VerifyRepeating(string input, int repeatCount, string output)
             }
         }
 
+        [Fact]
+        public void InvalidInput_Null()
+        {
+            using (HashAlgorithm hash = Create())
+            {
+                Assert.Throws<ArgumentNullException>(() => hash.ComputeHash((byte[])null));
+                Assert.Throws<ArgumentNullException>(() => hash.ComputeHash(null, 0, 0));
```

Comment:
Nit: for Argument*Exceptions, you can specify a first argument to the Throws call that is the name of the expected parameter.